src/pages/ExamsPage/components/AllowanceListActions.jsx

```diff
@@ -77,6 +211,7 @@ const AllowanceListActions = ({ allowance }) => {
         size="sm"
         data-testid="delete-allowance-icon"
       />
+      {EditModal(isEditModalOpen, setEditModalClosed, allowance, formatMessage)}
```

Comment:
this is more of a NIT but we're following the same pattern as delete except for passing the edit/delete handler which means you can't make edit stateless/functional like delete. I don't think it _needs_ to be functional but it would be nice for the sake of consistency.

Reply:
I see. So would I need to move the const and rendering of the edit to the modal to be inside of the AllowanceListActions, instead of being called before, as with the DeleteModal? Just want to make sure I'm understanding correctly.